src/style.css

```diff
@@ -1 +1,66 @@
-/* Styles go here */
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap");
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  width: 100%;
+  min-height: 100vh;
+  display: flex;
+  justify-content: center;
+  align-items: flex-start;
+  font-family: "Roboto", sans-serif;
+  margin: 0;
+}
+
+.header {
+  width: 1200px;
+  height: 60px;
+}
+
+.header .main-nav {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.main-nav-list {
+  list-style: none;
+
+}
+
+.header .main-nav .main-nav-list {
+  display: flex;
+  gap: 20px;
+}
+
+.logo {
+  /* display: flex; */
+  padding: 10px 50px;
+}
+
+.links {
+  /* display: flex; */
+  position: relative;
+  text-decoration: none;
+  color: black;
+  padding: 9px 0;
+  font-size: 12px;
+  text-transform: uppercase;
+}
```

Comment:
have class for `li` tag and shift padding and margins there , and only stylling to links should be here